nixos/release.nix

```diff
@@ -92,15 +92,33 @@ let
   }).config));
 
 
+  cleanConfig = import ./lib/eval-config.nix {
```

Comment:
We can measure once there's a working patch.

I'm struggling to make `evalModules` use `pkgs` that I pass it... just adding `pkgs` attribute to `args` parameter doesn't work, though I would think it should work.